opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -82,16 +82,16 @@ class DefaultAggregation(_AggregationFactory):
     This aggregation will create an actual aggregation depending on the
     instrument type, as specified next:
 
-    ========================= ====================================
-    Instrument                Aggregation
-    ========================= ====================================
-    `Counter`                 `SumAggregation`
-    `UpDownCounter`           `SumAggregation`
-    `ObservableCounter`       `SumAggregation`
-    `ObservableUpDownCounter` `SumAggregation`
-    `Histogram`               `ExplicitBucketHistogramAggregation`
-    `ObservableGauge`         `LastValueAggregation`
-    ========================= ====================================
+    ============================================= ====================================
+    Instrument                                    Aggregation
+    ============================================= ====================================
+    `Counter`                                     `SumAggregation`
+    `UpDownCounter`                               `SumAggregation`
+    `ObservableCounter`                           `SumAggregation`
+    `ObservableUpDownCounter`                     `SumAggregation`
+    `opentelemetry._metrics.instrument.Histogram` `ExplicitBucketHistogramAggregation`
```

Comment:
Why only use the full path for this instrument?

Reply:
because the class `instrument.Histogram` conflicts with `point.Histogram`